README.md

```diff
@@ -31,7 +31,6 @@ transfer an asset in a more realistic transfer scenario.
 | [State-Based Endorsement](asset-transfer-sbe) | This sample demonstrates how to override the chaincode-level endorsement policy to set endorsement policies at the key-level (data/asset level). | [Using State-based endorsement](https://github.com/hyperledger/fabric-samples/tree/master/asset-transfer-sbe) | Java, TypeScript | JavaScript |
```

Comment:
Should we add the transfer-events sample to this series? It wasn't quite sure if it fit.

Reply:
Yes, I've just merged a PR that adds the asset-transfer-events readme, we can add it to the root readme in a subsequent PR.